src/index.css

```diff
@@ -85,3 +85,91 @@
 .columnSort:hover {
   @apply text-controlGreyHover;
 }
+
+:root {
+  --rmdp-primary-green: #40bfbf;
+  --rmdp-secondary-green: #86efac;
+  --rmdp-shadow-green: #40bfbf;
+  --rmdp-today-green: #40bfbf;
+  --rmdp-hover-green: #40bfbf;
+  --rmdp-deselect-green: #40bfbf;
+  --rmdp-range-hover-green: #d8f2ea;
+}
+
+.green .rmdp-wrapper {
+  border: 1px solid var(--rmdp-secondary-green);
+  box-shadow: 0 0 5px var(--rmdp-secondary-green);
+}
+
+.green .rmdp-panel-body li {
+  background-color: var(--rmdp-primary-green);
+  box-shadow: 0 0 2px var(--rmdp-secondary-green);
+}
+
+.green .rmdp-week-day {
+  color: var(--rmdp-primary-green);
+}
+
+.green .rmdp-day.rmdp-deactive {
+  color: var(--rmdp-secondary-green);
+}
+
+.green .rmdp-range {
+  background-color: var(--rmdp-primary-green);
+  box-shadow: 0 0 3px var(--rmdp-shadow-green);
+}
+
+.green .rmdp-range-hover {
+  color: #73808c;
+  background-color: var(--rmdp-range-hover-green);
+  /* box-shadow: 0 0 3px var(--rmdp-shadow-green); */
+}
+
+.green .rmdp-arrow {
+  border: solid var(--rmdp-primary-green);
+  border-width: 0 2px 2px 0;
+}
+
+.green .rmdp-arrow-container:hover {
+  background-color: var(--rmdp-primary-green);
+  box-shadow: 0 0 3px var(--rmdp-secondary-green);
+}
+
+.green .rmdp-panel-body::-webkit-scrollbar-thumb {
+  background: var(--rmdp-primary-green);
+}
+
+.green .rmdp-day.rmdp-today span {
+  background-color: var(--rmdp-today-green);
+}
+
+.green .rmdp-rtl .rmdp-panel {
+  border-left: unset;
+  border-right: 1px solid var(--rmdp-secondary-green);
+}
+
+.green .rmdp-day.rmdp-selected span:not(.highlight) {
+  background-color: var(--rmdp-primary-green);
+  box-shadow: 0 0 3px var(--rmdp-shadow-green);
+}
+
+.green .rmdp-day:not(.rmdp-day-hidden) span:hover {
+  background-color: var(--rmdp-hover-green) !important;
+}
+
+.green .b-deselect {
+  color: var(--rmdp-deselect-green);
+  background-color: white;
+}
+
+.green .rmdp-action-button {
+  color: var(--rmdp-primary-green);
+}
+
+.green .rmdp-button:not(.rmdp-action-button) {
+  background-color: var(--rmdp-primary-green);
+}
+
+.green .rmdp-button:not(.rmdp-action-button):hover {
+  background-color: var(--rmdp-deselect-green);
+}
```

Comment:
Can we move this into a different `.css` file which indicates that the styles are dedicated for the calendar?

Something like `DateRangeInput.css` and move it inside of the `DateRangePicker` folder.